src/skai/model/docker_instructions.py

```diff
@@ -55,22 +56,7 @@ def get_docker_instructions(accelerator: str) -> tuple[str, list[str]]:
     # https://cloud.google.com/deep-learning-containers/docs/choosing-container
     base_image = GPU_BASE_IMAGE
     docker_instructions = [
-        # Add deadsnakes repo
-        'RUN apt update',
-        'RUN apt-get install software-properties-common -y',
-        'RUN add-apt-repository ppa:deadsnakes/ppa -y',
-
-        # Install Python 3.10',
-        'RUN apt update && apt install -y python3.10 python3.10-distutils',
-        'RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10',
-
-        # Replace python shell with python3.10',
-        'RUN unlink /usr/bin/python',
-        'RUN ln -s /usr/bin/python3.10 /usr/bin/python',
-
-        'RUN python -m pip install --pre --extra-index-url ' +
-        'https://developer.download.nvidia.com/compute/redist/jp/v50 ' +
-        'tensorflow==2.13'
+      'RUN apt-get update && apt-get install -y libcairo2-dev libjpeg-dev libgif-dev'
```

Comment:
There should be 2 more spaces here.

Reply:
Thank you, I have changed the spacing to 4.